### ISSUE TITLE
Bugs de Buffs

### ISSUE THREAD

**Issue by ghost** [RoDNet/Bugtracker-Infinium-4.0.6a#1240]
Bueno, el bug es que si por ejemplo, le pones un buff a un pj de lvl bajo, le da los stats como si fuera del level que lo pone.. ejemplo, yo soy 85, le pongo "palabra de poder: entereza", a un lvl 15, en vez de dar los puntos (como 20p de aguante aproximado), le da 525p de aguante.

Como se puede ver en la siguiente imagen, esto pasa con todos los buff que no estan en porcentaje.

![prueba](https://f.cloud.github.com/assets/3508051/599719/5f89646a-cc56-11e2-92a0-3b7ec8af5ec6.jpg)

(censuro la identidad del pj, porque tome la imagen para ponerla aquí sin su consentimiento xd).

Dejo dos enlaces con skills que probe:
http://es.wowhead.com/spell=21562
http://old.wowhead.com/spell=27683

**Comment by DatJetto** [comment]
Pasa con algunas spells, hay otras que directamente dice El nivel del personaje es muy bajo o cosas así.

**Comment by ghost** [comment]
Claro, esas spells que mencionas son así, pero estas (buffos) no escalan con el nivel del pj al que lo pones como debería ser.

**Comment by DatJetto** [comment]
Eso era obvio.
